### PR TITLE
Create de

### PR DESCRIPTION
**Pull request by Coregenetic**
deutsch hinzugefügt

- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest)

> **Yes**

- Does your code fit the style guidelines? [yes/no]

> **Yes**

- Does your PR fit the contribution guidelines? [yes/no]

> **Yes**

### CODE REVIEW

**Review by GodyFromDiscord — CHANGES_REQUESTED** [review]
Please rename the file to de.lua and not just de

File: locales/de [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        pacific_robbery_alert = "Pacific Standard Bank robbery attempt",
+        break_safe_open_option_target = "Tresor öffnen",
+        break_safe_open_option_drawtext = "[E] Den Tresor aufbrechen",
+        validating_bankcard = "Karte validieren...",
+        thermite_detonating_in_seconds = "Thermit explodiert in %{time} Sekunde(n)",
+        bank_robbery_police_call = "10-90: Bank Raub"
+    }
+}
+
+Lang = Lang or Locale:new({
```
Comment (GodyFromDiscord) [inline locales/de:33]:
```
if GetConvar('qb_locale', 'en') == 'de' then
    Lang = Locale:new({
        phrases = Translations,
        warnOnMissing = true,
        fallbackLang = Lang,
    })
end
```


Please add this due to the locale system

**Review by GodyFromDiscord — CHANGES_REQUESTED** [review]
Update requested changes

**Comment by ChristianBDev** [comment]
@Coregenetic Please Update to the requested changes so we can continue

**Pull request closed without merging**